terraso_backend/config/settings.py

```diff
@@ -364,3 +365,17 @@
     "/healthz/",
     "/web-client/sitemap.xml",
 ]
+
+
+class JWTProvider(TypedDict):
+    url: str
+    client_id: str
+    name: str
+
```

Comment:
I made this is a type hint, so that it's obvious to users what type to use for the mobile settings. I think it might be a good idea to add a comment explaining that.